cicecore/cicedynB/dynamics/ice_dyn_evp.F90

```diff
@@ -1435,15 +1475,16 @@ subroutine stress_U   (nx_block,   ny_block,  &
                              uvelU,      vvelU,     &
                              dxE,        dyN,       &
                              dxU,        dyU,       &
+                             tarea,                 &
                              ratiodxN,   ratiodxNr, &
                              ratiodyE,   ratiodyEr, &
-                             epm,  npm,  uvm,       &
+                             epm,  npm, hm, uvm,    &
```

Comment:
I think it would be preferable to restrict ourselves to 2 arguments per line, for better readability.